Packs/QuestKace/Integrations/QuestKace/QuestKace.py

```diff
@@ -0,0 +1,666 @@
+           Returns:
+               Response from API.
+        """
+        headers = {
+            'Accept': 'application/json',
+            'x-dell-csrf-token': self._token,
+            'x-dell-api-version': '5',
+            'Cookie': self._cookie,
+            'Content-Type': 'application/json'
+        }
+        return self._http_request("POST", url_suffix="/service_desk/tickets", headers=headers, data=data)
+
+    def update_ticket_request(self, ticket_id: str, data: str) -> dict:
+        """Update Ticket.
+           Returns:
+               Response from API.
+        """
+        headers = {
+            'Accept': 'application/json',
+            'x-dell-csrf-token': self._token,
+            'x-dell-api-version': '5',
+            'Cookie': self._cookie,
+            'Content-Type': 'application/json'
+        }
+        return self._http_request("POST", url_suffix=f"/service_desk/tickets/{ticket_id}", headers=headers, data=data)
+
+    def delete_ticket_request(self, ticket_id: str) -> dict:
+        """Delete Ticket.
+           Returns:
+               Response from API.
+        """
+        headers = {
+            'Accept': 'application/json',
+            'x-dell-csrf-token': self._token,
+            'x-dell-api-version': '5',
+            'Cookie': self._cookie,
+            'Content-Type': 'application/json'
+        }
+        return self._http_request("DELETE", url_suffix=f"/service_desk/tickets/{ticket_id}", headers=headers)
+
+    def ticket_by_id_request(self, id: int) -> dict:
+        """Specific ticket details by ID.
+            Args:
+                id: id for filtering by it.
+           Returns:
+               Response from API.
+        """
+        headers = {
+            'Accept': 'application/json',
+            'x-dell-csrf-token': self._token,
+            'x-dell-api-version': '5',
+            'Cookie': self._cookie
+        }
+        filter_fields = f"id eq {id}"
+        return self._http_request("GET", url_suffix=f"/service_desk/tickets?filtering={filter_fields}", headers=headers)
+
+
+def test_module(client: Client, args=None) -> Tuple[str, dict, dict]:
+    """Function which checks if there is a connection with the api.
+        Args:
+            client : Integretion client which communicates with the api.
+            args: Users arguments of the command.
+       Returns:
+           human readable, context, raw response of this command.
+    """
+    response = client.tickets_list_request()
+    list_tickets_res = response.get('Tickets')
+    if list_tickets_res and demisto.params().get('isFetch'):
+        parsed_time = (datetime.utcnow() - timedelta(days=20))
+        incidents, parsed_time = fetch_incidents_create_incidents(list_tickets_res, "1", '%Y-%m-%dT%H:%M:%SZ', parsed_time)
+        for incident in incidents:
+            if not all(key in incident for key in ('name', 'occurred', 'rawJSON')):
+                raise DemistoException('Missing params in fetch incidents Test.')
```

Comment:
not sure this is needed. you can just:
_, _ = fetch_incidents_create_incidents(list_tickets_res, "1", '%Y-%m-%dT%H:%M:%SZ', parsed_time)

and if it passed, than you can return 'ok'